domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/base/ErrorNotice.java

```diff
@@ -70,6 +70,7 @@
     protected static final int E_049 = 49;
     protected static final int E_050 = 50;
     protected static final int E_051 = 51;
+    public static final int E_052 = 52;
```

Comment:
This is `public` so the detected error type in `GeospatialUtilsImplTest` can be asserted, which is in the `adapter` module.

Reply:
I think that we should add in-line comments to explain why this error code is public instead of protected :)